### ISSUE TITLE
Some http redirection cause metalink4 XML parse error in v0.8a4

### ISSUE THREAD

**Issue by miurahr** [miurahr/aqtinstall#110]
With v0.8a4 CI got connection error.

```
2020-03-06T04:35:44.6523119Z 2020-03-06 04:35:44,651 - aqt - ERROR - Unexpected meta4 file; parse error: not well-formed (invalid token): line 1, column 2
2020-03-06T04:35:44.6537454Z 2020-03-06 04:35:44,653 - aqt - INFO - Redirected to new URL: http://mirrors.geekpie.club/qtproject/online/qtsdkrepository/windows_x86/desktop/qt5_5132/qt.qt5.5132.win64_mingw73/5.13.2-0-201910281254qtxmlpatterns-Windows-Windows_10-Mingw73-Windows-Windows_10-X86_64.7z
2020-03-06T04:35:47.8838460Z 2020-03-06 04:35:47,865 - aqt - ERROR - Unexpected meta4 file; parse error: not well-formed (invalid token): line 1, column 2
2020-03-06T04:35:47.8878332Z 2020-03-06 04:35:47,865 - aqt - INFO - Redirected to new URL: http://mirrors.geekpie.club/qtproject/online/qtsdkrepository/windows_x86/desktop/qt5_5132/qt.qt5.5132.win64_mingw73/5.13.2-0-201910281254qtscxml-Windows-Windows_10-Mingw73-Windows-Windows_10-X86_64.7z

2020-03-06T04:36:08.1236079Z 2020-03-06 04:36:08,122 - aqt - ERROR - Connection error: HTTPConnectionPool(host='mirrors.geekpie.club', port=80): Max retries exceeded with url: /qtproject/online/qtsdkrepository/windows_x86/desktop/qt5_5132/qt.qt5.5132.win64_mingw73/5.13.2-0-201910281254qtxmlpatterns-Windows-Windows_10-Mingw73-Windows-Windows_10-X86_64.7z (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000029EB02D7B48>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))
2020-03-06T04:36:08.1238613Z 2020-03-06 04:36:08,122 - aqt - ERROR - Installation error detected.
```

It seems that http://download.qt.io/ server responses  broken or wrong metalink4 xml file with redirect location become blacklisted site 'http://mirrors.geekpie.club'
 altlink function can not determine another mirror then it returns original black site.

**Comment by miurahr** [comment]
We can set fall back white site for the case when failing to get metalink data.

**Comment by miurahr** [comment]
Fix is released in v0.8b1
